connector/src/main/resources/config.yml

```diff
@@ -79,8 +79,8 @@ floodgate-key-file: key.pem
 #    password: javaccountpassword123 # Your Minecraft: Java Edition password
 #    microsoft-account: true # Whether the account is a Mojang or Microsoft account.
 #
-#  bluerkelp2: 
-#    email: not_really_my_email_address_mr_minecrafter53267@gmail.com 
+#  bluerkelp2:
```

Comment:
Please revert this diff.

Reply:
I see, that I changed that, but why should it have spaces at the end?

Reply:
IntelliJ IDEA really dont like me doing that... It always removes one of them before commit :D

Reply:
```suggestion
#  bluerkelp2:
```